source/production/F0.Minesweeper.Components/Logic/Cell/CellVisualisationManager.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Abstractions.Enums;
+
+namespace F0.Minesweeper.Components.Logic.Cell
+{
+	internal class CellVisualisationManager : ICellVisualisationManager
```

Comment:
`Visualization` vs `Visualisation`?
Do we go for US-American or British English?
(If we change to US-English, please also rename both `GetVisualisation` methods and the `CellVisualisationManagerTests` type/file)

Reply:
Personally I always use British English but since this really is a matter of taste lets discuss this in the next session!

Reply:
We may overrule you here with US-English.

Reply:
Changed every `Visualisation` I could find to `Visualization`